client/src/styles/legacy/_style.scss

```diff
@@ -687,6 +687,25 @@ html, body {
   }
 }
 
+.validationerror .panel--padded .ui-tabs-nav {
```

Comment:
CSS is in the file (which is a bit of a dumping ground) because `.panel-padding .ui-tabs-nav` is directly above this